src/server/routes/v1/workshops/[slug].get.ts

```diff
@@ -0,0 +1,8 @@
+import { defineEventHandler, getRouterParam } from 'h3';
+import { WORKSHOP } from '../workshops';
+
+export default defineEventHandler((event) => {
+  const slug = getRouterParam(event, 'slug');
+  if (!slug) return null;
```

Comment:
A possible alternative could be to add some filter to the same workshop API, or filter directly in the `[workshops/[slug].page.ts](https://github.com/ngromeconf/ngrome-website/pull/76/files#diff-bfafb1a2c6105d61b7d92829ec5b1379c9fc7ad8341a5fe7c62adb7834fe806b)` (which could be not the optimal solution ) but should work!